### PR TITLE
Add basic nf-core documentation

### PR DESCRIPTION
**Pull request by scwatts**

### CODE REVIEW

**Comment by scwatts** [comment]
Force pushing after rebase on top of `dev`

**Comment by scwatts** [comment]
Force pushed to rebase on top of `dev` after merging latest template

**Comment by scwatts** [comment]
Another force push to rebase on top of `dev`

**Review by charlesshale — APPROVED** [review]

File: docs/usage.md [hunk 1]
```diff
@@ -6,61 +6,143 @@
...
+PURPLE data as the starting point and skip variant calling processes. Additionally, reference/resource files can be
+staged locally to optimise execution or modified to create user-defined driver gene panels.
+
+> [!WARNING]
+> There are important requirements when using BAMs as input instead of FASTQs:
+>
+> - STAR must have been run with [specific
+>   parameters](https://github.com/hartwigmedical/hmftools/tree/master/isofox#a-note-on-alignment-and-multi-mapping),
+>   this is critical for WTS data, and
+> - reads are expected to have been aligned to one of the Hartwig-distributed reference genomes (user-defined genomes may be used though are not recommended)
```
Comment (casslitch) [inline docs/usage.md:30]: Could add a link here to a page outlining which files/params would need to be changed in order run the pipeline from a user-defined genome

File: docs/usage.md [hunk 2]
```diff
@@ -6,61 +6,143 @@
...
+P1__wgts,P1,SC,tumor,rna,fastq,library_id:SC_library;lane:001,/path/to/P1.SC.tumor.rna.wts.001.R1.fastq.gz;/path/to/P1.SC.tumor.rna.wts.001.R2.fastq.gz
 ```
 
-### Multiple runs of the same sample
+#### BAM
 
-The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
+> [!NOTE]
+> Inputs with the `bam` filetype will be processed by MarkDups as required by hmftools. Where an input BAM has already
+> been processed by MarkDups, you can avoid needless reprocessing by setting `bam_markdups` as the filetype instead.
```
Comment (casslitch) [inline docs/usage.md:74]: Change 'MarkDups' to 'HMF MarkDups' (if it's important that HMF MarkDups has been run and not some other duplicate marking tool)

**Review by casslitch — APPROVED** [review]

**Pull request merged**